lgsm/functions/command_install_init.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```

Comment:
please use the normal bash header `#!/bin/bash` instead and add the default header for the description of the file.

Reply:
fixed